x-pack/plugins/ml/public/file_datavisualizer/components/fields_stats/fields_stats.js

```diff
@@ -79,7 +79,16 @@ function createFields(results) {
           }
         }
 
-        field.percent = ((field.count / numMessagesAnalyzed) * 100);
+        const percent = ((field.count / numMessagesAnalyzed) * 100);
+        field.percent = Math.round(percent * 100) / 100;
```

Comment:
`Math.round(<value> * 100) / 100;` could become a helper function, something like `getValueFormattedForDisplay()` or similar.

Reply:
latest commit covers this